test/test-subsystem/TESTSubsystemBinary.cpp

```diff
@@ -59,9 +61,13 @@ std::string CTESTSubsystemBinary::toString(const void* pvValue, size_t size) con
 
 void CTESTSubsystemBinary::fromString(const std::string& strValue, void* pvValue, size_t size)
 {
-    uint32_t uiValue = strtoul(strValue.c_str(), NULL, 0);
+    uint32_t uiValue;
 
     assert(size <= sizeof(uiValue));
 
-    memcpy(pvValue, (const void*)&uiValue, size);
+    if (!convertTo(strValue, uiValue)) {
+        throw std::string("Unable to convert \"") + strValue + std::string("\" to uint32");
```

Comment:
That is dangerous, the PF is not exception safe. Where is the matching catch block ?